swift/llm/utils/argument.py

```diff
@@ -627,6 +627,11 @@ class ExportArguments(InferArguments):
     commit_message: str = 'update files'
 
 
+    def __post_init__(self) -> None:
```

Comment:
Here, the dataset registration can be removed. It is not necessary to register the dataset again, as it will be registered in the InferArguments.